locale/ar_IQ/api.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+
+<!--
+  * locale/en_US/api.xml
```

Comment:
This should be .../ar_IQ/..., but I will fix this later.

Reply:
Please do. I have some problems using online editor of Github.